path/tree_test.go

```diff
@@ -78,10 +78,13 @@ func TestWildcardTree(t *testing.T) {
 	}
 	for _, test := range find {
 		path := strings.Split(test.path, "/")
-		got := tree.Find(path, 0)
+		got, matched := tree.Find(path, 0)
 		if test.value != got {
 			t.Errorf("for: %v, want: %v, got: %v", test.path, test.value, got)
 		}
+		if s := strings.Join(matched, "/"); s != test.path {
```

Comment:
This is the added check to test the returned matchedPath. There are checks added in other tests too.